plugins/source/aws/client/spec.go

```diff
@@ -14,6 +14,8 @@ type Account struct {
 
 type AwsOrg struct {
 	OrganizationUnits           []string `json:"organization_units,omitempty"`
+	SkipAccounts                []string `json:"skip_accounts,omitempty"`
+	SkipOrganizationalUnits     []string `json:"skip_organization_units,omitempty"`
```

Comment:
It can also be `r-<ID>` for root OUs. I found a regex for OU patterns here though https://docs.aws.amazon.com/organizations/latest/APIReference/API_UpdateOrganizationalUnit.html which I'm now using in conjunction with the undocumented `r-<ID>` case. Hopefully there aren't other undocumented cases